components/prism-bqn.js

```diff
@@ -0,0 +1,61 @@
+Prism.languages.bqn = {
+	'shebang': {
+	pattern: /^#!\s*\/.*/,
```

Comment:
`\s` includes new lines, which is probably incorrect.
```suggestion
	pattern: /^#![ \t]*\/.*/,
```